vmdb/app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -10,6 +10,9 @@ def textual_group_properties
 
   def textual_group_relationships
     items = %w(container_nodes container_services container_groups)
+    if @ems.class.ems_type.eql? 'openshift'
+      items.concat(%w(container_routes container_projects))
```

Comment:
Actually, even further, at the top level here, you don't even need the conditional.  Instead the individual line item methods below could possible `return nil if @ems.kind_of?(EmsOpenstack)`.  Then you get the logic living with the individual methods.